app/models/transformation_mapping.rb

```diff
@@ -1,9 +1,115 @@
 class TransformationMapping < ApplicationRecord
+  VM_CONFLICT = N_("Conflict").freeze
+  VM_NON_EXIST = N_("VM does not exist").freeze
+  VM_VALID = N_("OK").freeze
```

Comment:
You do not want to set this as a constant. It will get set once with whatever localization is in place when the model loads. You are trying to honor the current users local which is dynamic.

I mentioned the previous PR here https://github.com/ManageIQ/manageiq/pull/17251#pullrequestreview-109479263

Reply:
```N_("OK")``` does not translate, but tells the parser: ```"OK"``` needs translation.

Reply:
@AparnaKarve @h-kataria Do you know how we are handling this in other areas of the application where the backend is sending strings to the UI through the API?

Reply:
@mzazrivec any input on this?

Reply:
@gmcculloug - sorry for the delayed response (somehow it just missed my radar)

Some of the other areas that I know are handling it exactly as @bzwei and @lfu have implemented it above.

Like @lfu said, the `N_` prefix just tells the i18n rake task to pick up the string for translation, but it's the client that is responsible for the actual translation/string substitution.

@mzazrivec Please validate the above. Thanks.

Reply:
Maybe these should be single words, i.e. like task statuses, "ok", "missing", "conflict", i.e. presentation independent.

Reply:
This is how we display the status in the UI -

<img width="814" alt="screen shot 2018-04-19 at 8 15 26 am" src="https://user-images.githubusercontent.com/1538216/39001141-71f77cce-43aa-11e8-8526-4ad57e74e71e.png">

Single words/minimal words are fine, but they should convey the exact status.
The status shown in the screenshot - 
`VM has been added to another plan` is particularly quite helpful.

@lfu @bzwei In the screenshot above, the csv had a blank entry for the `name` field in row1.
For a case like this, is it possible to specify a different reason there, something like - 
`VM could not be determined`/`undeterminable`